doc/1-intro/README.md

```diff
@@ -27,3 +27,5 @@ once familiar with the tracker, look to [9-guides](../9-guides/README.md) for us
 ## tutorial?
 
 [Furnace Tutorials](https://youtube.com/playlist?list=PLCELB6AsTZUnwv0PC5AAGHjvg47F44YQ1): video tutorials created by Spinning Square Waves. be noted that these may not apply to the current version.
+
+[How to Learn Chiptune Trackers](https://www.youtube.com/watch?v=Q37XuOLz0jw): video tutorial created by Button Masher. covers general concepts using Furnace for demonstration.
```

Comment:
Put it above the previous item. Also, rephrase it. "Covers general concepts" is not so clear.